crocket/src/main/java/com/crocket/view/LevelView.java

```diff
@@ -72,11 +75,34 @@ private JLabel makeLabel(DrawableEntity entity, BufferedImage bufferedImage) {
         return label;
     }
     
-    // TODO: FIX THIS!!!!
     private void drawDirectionLine(DrawableEntity entity) {
-     
+        DrawDirectionLine panel = new DrawDirectionLine();
+        panel.setBounds(entity.getxPosition()-panel.getWidth()/2 +10, entity.getyPosition()-panel.getHeight()/2 +10, panel.getWidth(), panel.getHeight());
```

Comment:
Where do we get 10 from? Is this something that could be a constant?

Reply:
fixed?